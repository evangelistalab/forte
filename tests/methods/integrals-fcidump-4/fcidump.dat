molecule {
-1 2
H 0.0 0.0 0.0
H 0.0 0.0 1.0
H 0.0 2.0 0.0
H 0.0 2.0 1.0
units bohr
symmetry c1
}

set {
  reference rohf
  basis sto-3g
  scf_type pk
  e_convergence 14
}

e, wfn = energy('fci',return_wfn=True)
fcidump(wfn)
