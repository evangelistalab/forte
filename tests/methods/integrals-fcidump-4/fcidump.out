
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 undefined 

                         Git: Rev {fcidump_sym} 00c99be dirty


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
    (doi: 10.1021/acs.jctc.7b00174)


                         Additional Contributions by
    P. Kraus, H. Kruse, M. H. Lechner, M. C. Schieber, R. A. Shaw,
    A. Alenaizan, R. Galvelis, Z. L. Glick, S. Lehtola, and J. P. Misiewicz

    -----------------------------------------------------------------------


    Psi4 started on: Monday, 06 July 2020 09:50PM

    Process ID: 98680
    Host:       CHE-FEVANGE-01I (2)
    PSIDATADIR: /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4
    Memory:     500.0 MiB
    Threads:    1
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule {
-1 2
H 0.0 0.0 0.0
H 0.0 0.0 1.0
H 0.0 2.0 0.0
H 0.0 2.0 1.0
units bohr
symmetry c1
}

set {
  reference rohf
  basis sto-3g
  scf_type pk
  e_convergence 14
}

e, wfn = energy('fci',return_wfn=True)
fcidump(wfn)
--------------------------------------------------------------------------

Scratch directory: /tmp/

*** tstart() called on CHE-FEVANGE-01I (2)
*** at Mon Jul  6 21:50:26 2020

   => Loading Basis Set <=

    Name: STO-3G
    Role: ORBITAL
    Keyword: BASIS
    atoms 1-4 entry H          line    19 file /Users/fevange/Source/psi4/objdir-Debug/stage/share/psi4/basis/sto-3g.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                             ROHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: D2h

    Geometry (in Bohr), charge = -1, multiplicity = 2:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         H            0.000000000000    -1.000000000000    -0.500000000000     1.007825032230
         H            0.000000000000    -1.000000000000     0.500000000000     1.007825032230
         H            0.000000000000     1.000000000000    -0.500000000000     1.007825032230
         H            0.000000000000     1.000000000000     0.500000000000     1.007825032230

  Running in c1 symmetry.

  Rotational constants: A =     59.73228  B =     14.93307  C =     11.94646 [cm^-1]
  Rotational constants: A = 1790728.69418  B = 447682.17355  C = 358145.73884 [MHz]
  Nuclear repulsion =    3.894427190999916

  Charge       = -1
  Multiplicity = 2
  Electrons    = 5
  Nalpha       = 3
  Nbeta        = 2

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-14
  Density threshold  = 1.00e-08
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: STO-3G
    Blend: STO-3G
    Number of shells: 4
    Number of basis function: 4
    Number of Cartesian functions: 4
    Spherical Harmonics?: true
    Max angular momentum: 0

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   4
      Number of AO shells:               4
      Number of primitives:             12
      Number of atomic orbitals:         4
      Number of basis functions:         4

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 110 doubles for integral storage.
  We computed 55 shell quartets total.
  Whereas there are 55 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 1.3613945736E-01.
  Reciprocal condition number of the overlap matrix is 5.1279114618E-02.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF (no occupation information).

   -------------------------
    Irrep   Nso     Nmo    
   -------------------------
     A          4       4 
   -------------------------
    Total       4       4
   -------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @ROHF iter SAD:    -1.78069493638582   -1.78069e+00   0.00000e+00 
   @ROHF iter   1:    -1.23857246047707    5.42122e-01   4.64449e-16 DIIS
   @ROHF iter   2:    -1.23857246047707   -1.77636e-15   1.48088e-16 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A     -0.374773     2A      0.056242  

    Singly Occupied:                                                      

       3A      0.939387  

    Virtual:                                                              

       4A      1.872780  

    Final Occupation by Irrep:
              A 
    DOCC [     2 ]
    SOCC [     1 ]

  @ROHF Final Energy:    -1.23857246047707

   => Energetics <=

    Nuclear Repulsion Energy =              3.8944271909999157
    One-Electron Energy =                 -10.2855109785431083
    Two-Electron Energy =                   5.1525113270661205
    Total Energy =                         -1.2385724604770720

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:     0.0000

  Electronic Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:    -0.0000

  Dipole Moment: [e a0]
     X:     0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000

  Dipole Moment: [D]
     X:     0.0000      Y:     0.0000      Z:    -0.0000     Total:     0.0000


*** tstop() called on CHE-FEVANGE-01I (2) at Mon Jul  6 21:50:26 2020
Module time:
	user time   =       0.31 seconds =       0.01 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.31 seconds =       0.01 minutes
	system time =       0.01 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of threads:                 1
      Number of atoms:                   4
      Number of AO shells:               4
      Number of SO shells:               4
      Number of primitives:             12
      Number of atomic orbitals:         4
      Number of basis functions:         4

      Number of irreps:                  1
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [   4 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 55 non-zero two-electron integrals.
        Stored in file 33.


         ---------------------------------------------------------
                          Configuration Interaction
                            (a 'D E T C I' module)

                 C. David Sherrill, Daniel G. A. Smith, and
                              Matt L. Leininger
         ---------------------------------------------------------

Warning: iopen=0,opentype!=closed. Making iopen=1

   ==> Parameters <==

    EX LEVEL       =        5      H0 BLOCKSIZE  =     1000
    VAL EX LEVEL   =        0      H0 GUESS SIZE =     1000
    H0COUPLINGSIZE =        0      H0 COUPLING   =       NO
    MAXITER        =       24      NUM PRINT     =       20
    NUM ROOTS      =        1      ICORE         =        1
    PRINT LVL      =        1      FCI           =      YES
    R CONV         = 1.00e-04      MIXED         =      YES
    E CONV         = 1.00e-14      MIXED4        =      YES
    R4S            =       NO      REPL OTF      =       NO
    DIAG METHOD    =      SEM      FOLLOW ROOT   =        0
    PRECONDITIONER = DAVIDSON      UPDATE        = DAVIDSON
    S              =   0.5000      Ms0           =       NO
    GUESS VECTOR   =  H0BLOCK      OPENTYPE      = HIGHSPIN
    COLLAPSE SIZE  =        1      HD AVG        = EVANGELISTI
    MAX NUM VECS   =       25      REF SYM       =     AUTO
    IOPEN        =      YES

    EX ALLOW       =  1  1  1  1  1 
    STATE AVERAGE  =  0(1.00) 

   ==> CI Orbital and Space information <==

   ------------------------------------
               Space    Total     A
   ------------------------------------
                 Nso        4     4
                 Nmo        4     4
               Ndocc        2     2
               Nsocc        1     1
   ------------------------------------
              CI Spaces
   ------------------------------------
        Dropped DOCC        0     0
              Active        4     4
        Dropped UOCC        0     0
   ------------------------------------

   ==> Setting up CI strings <==

    There are 4 alpha and 6 beta strings
    The CI space requires 24 (2.40E+01) determinants and 1 blocks

   ==> Transforming CI integrals <==

	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Transforming the one-electron integrals and constructing Fock matrices
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.

   ==> Starting CI iterations <==

    H0 Block Eigenvalue =  -1.25289369

    Simultaneous Expansion Method (Block Davidson Method)
    Using 1 initial trial vectors

     Iter   Root       Total Energy       Delta E      C RMS

   @CI  0:     0     -1.252893688642   -5.1473E+00   2.0782E-15  
    Warning: Norm of correction (root 0) is < 1.0E-13
   @CI  1:     0     -1.252893688642   8.8818E-16   8.3024E-16 c

   ==> Energetics <==

    SCF energy =           -1.238572460477072
    Total CI energy =      -1.252893688642302

   ==> FCI root 0 information <==

    FCI Root 0 energy =    -1.252893688642302

   The 20 most important determinants:

    *   1    0.997252  (    0,    0)  1AX 2AX 3AA 
    *   2   -0.044220  (    2,    2)  1AA 2AB 3AX 4AA 
    *   3    0.043985  (    2,    3)  1AX 3AA 4AX 
    *   4   -0.031024  (    3,    1)  1AB 2AA 3AX 4AA 
    *   5    0.021472  (    3,    4)  2AX 3AA 4AX 
    *   6   -0.013197  (    0,    5)  1AA 2AA 3AX 4AB 

		 "A good bug is a dead bug" 

			 - Starship Troopers

		 "I didn't write FORTRAN.  That's the problem."

			 - Edward Valeev
Writing integrals in FCIDUMP format to INTDUMP
 MINTS: Wrapper to libmints.
   by Justin Turney

   Calculation information:
      Number of threads:                 1
      Number of atoms:                   4
      Number of AO shells:               4
      Number of SO shells:               4
      Number of primitives:             12
      Number of atomic orbitals:         4
      Number of basis functions:         4

      Number of irreps:                  1
      Integral cutoff                 0.00e+00
      Number of functions per irrep: [   4 ]

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

      Computing two-electron integrals...done
      Computed 55 non-zero two-electron integrals.
        Stored in file 33.

	Presorting SO-basis two-electron integrals.
	Sorting File: SO Ints (nn|nn) nbuckets = 1
	Transforming the one-electron integrals and constructing Fock matrices
	Starting first half-transformation.
	Sorting half-transformed integrals.
	First half integral transformation complete.
	Starting second half-transformation.
	Two-electron integral transformation complete.
Integral transformation complete!
Writing TEI integrals in FCIDUMP format to INTDUMP
Writing frozen core operator in FCIDUMP format to INTDUMP
Writing orbital energies in FCIDUMP format to INTDUMP
Writing frozen core + nuclear repulsion energy in FCIDUMP format to INTDUMP
Done generating INTDUMP with integrals in FCIDUMP format.

    Psi4 stopped on: Monday, 06 July 2020 09:50PM
    Psi4 wall time for execution: 0:00:00.47

*** Psi4 exiting successfully. Buy a developer a beer!
